src/visualization/types/Map/view.tsx

```diff
@@ -85,7 +82,7 @@ const GeoPlot: FC<Props> = ({result, properties}) => {
   useEffect(() => {
     try {
       setCoordinateError(RemoteDataState.Loading)
-      if (isBehindFlag) {
+      if (CLOUD) {
```

Comment:
I think we can completely remove this if else. The reason we are using this here and on line 146 and 192 is that we were trying to separate map 1.0 from map 2.0 (mapGeoOptions). So that the users wouldn't see any differences on their map rendering until we were ready to to turn on map 2.0